docs/src/explanations/connectable.md

```diff
@@ -8,32 +8,43 @@ section: "chisel3"
 
 ---
 
+## Terminology
+
+- "Chisel type" - a `Data` that is not bound to hardware
+- "component" - a `Data` that is bound to hardware (`IO`, `Reg`, `Wire`, etc.)
+- "member" - a child Chisel type/component of a parent Chisel type or component
+- "field" - a named member of a `Record` or `Bundle` Chisel type or component
```

Comment:
not sure i get the difference between `member` and `field`. Also what is the difference from that and `element` which is used in the code?

Maybe you could "use it in a sentence" and show an example of a Bundle and say "foo is a component", "foo.bar is a member", "bar is a field" ??

Reply:
yeah, I think member and element are similar?

Reply:
`Element` means `Non-Aggregate`, so I'm going to use the 'member' terminology.